### ISSUE TITLE
简介 HiTIN: Hierarchy-aware Tree Isomorphism Network forHierarchical Text Classification | Measure Zero

### ISSUE THREAD

**Issue by Shiina18** [Shiina18/shiina18.github.io#190]
https://shiina18.github.io/machine%20learning/2023/09/25/hitin/ 

 Zhu, H., Zhang, C., Huang, J., Wu, J., &amp; Xu, K. (2023). HiTIN: Hierarchy-aware Tree Isomorphism Network for Hierarchical Text Classification. arXiv preprint arXiv:2305.15182. (左上两个图) 先把原始的类别层级树结构转换成新的树结构 coding tree (作者自创概念). 获得文本表示之后 (上图右下 $H$), 转换成 coding tree 叶结点的 embedding (上图 $X_G$, 维度为 (叶结点数, 叶结点 embedding size)). (第一行第三个图) 然后用常规的 GNN, 在 coding tree 上, 从底向上, 递归地根据子结点的 embedding 获得结点的 embedding (子结点 embedding 先求和再 MLP). 最后把每一层的结点 embedding 做 pooling (上图右上 $H_T$), 拼接起来, 经过线性层得到最终 logits (标准的多标签分类). 主要创新点是上述第一步. 论文结果中, 相比其他针对层次分类的网络, 本网络参数少而且效果好很多. 重写的代码见 这里.
